README.txt

```diff
@@ -1 +1,7 @@
 https://fedorahosted.org/fedoracommunity/wiki/Development
+
+Note: Some more dependencies to be installed:-
+
+$ sudo dnf -q -y install python-webhelpers fedmsg
```

Comment:
@pypingou yeah i agree with your point, i have added these options just because these are used in bootstrap.py:
`run('sudo dnf install -q -y ' + ' '.join(reqs))`